ui/src/index.js

```diff
@@ -1,18 +1,22 @@
 import { history, InvenioILSApp } from '@inveniosoftware/react-invenio-app-ils';
 import React from 'react';
-import 'react-app-polyfill/ie11'; // For IE 11 support
 import ReactDOM from 'react-dom';
 import { OverridableContext } from 'react-overridable';
 import { Router } from 'react-router-dom';
 import 'semantic-ui-less/semantic.less';
 import { config } from './config';
 import { overriddenCmps } from './overridableMapping';
 
-ReactDOM.render(
-  <Router history={history}>
-    <OverridableContext.Provider value={overriddenCmps}>
-      <InvenioILSApp config={config} />
-    </OverridableContext.Provider>
-  </Router>,
-  document.getElementById('app')
-);
+// Checks if browser is IE (unsupported)
+const isIE = !!document.documentMode;
```

Comment:
Since the `index.js` is loaded anyway, I would suggest to put this in `react-invenio-app-ils` and export it to avoid duplications, otherwise everyone creating their own ILS will have to copy/paste this.
The idea is that here you will simply do:
```
import { IEFallback, history, InvenioILSApp } from '@inveniosoftware/react-invenio-app-ils';

...

if (!isIE) {
...
} else {
  IEFallback()
}
```

The IEFallback function contains all the JS  in the `check-for-IE` file. I would also add the CSS there, or even inline for this...
WDYT?